### ISSUE TITLE
Fileclass Fields Disappeared

### ISSUE THREAD

**Issue by ipxfer** [mdelobelle/metadatamenu#496]
I created three classes - one root and two subordinate.  All three classes were configured with unique fileclass fields, and was working well with templater in creating new notes.  I then opened up another Obsidian from different desktop (synced with GitHub), and found all three classes' fileclass fields blank.  Thinking it needs to sync up, but not the case.  Both desktops showed empty fileclass fields.  Anyone else have the same experience?

Thanks.

**Comment by malevans** [comment]
Yes - I am seeing the same. I note that there was an update a few days ago. Not sure if that is related or not. I am recreating now and will monitor

Ver 0.7.0-beta-1
Obs v1.5.3

**Comment by MrStuki** [comment]
I have the problem that the fields are still in the FileClass(Name).md. However, these are no longer recognised by the plugin.
